sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_model_tools/models.py

```diff
@@ -179,7 +183,7 @@ def __repr__(self):
 # ===========================================================
 
 
-class OpenAICompletionsModel(LLMBase):
+class OpenAICompletionsModel(LLMBase):  # pylint: disable=too-many-instance-attributes
```

Comment:
This class isn't used too. We inherited a lot of unused code that we can get rid of now.